### DISCUSSION TITLE
Periodic Tessellation

### DISCUSSION THREAD

**Discussion by majidkavousi** [neperfepx/neper#442] [Q&A] [answered]
Hi everyone

Is it possible to create a full periodic tessellation but force it to remain exactly as the specified domain? For instance, to have a periodic tessellation in 2D with an exact square shape. 

When I activated the periodicity, the domain changes according to the crystals in the boundary of the domain as below:
![image](https://user-images.githubusercontent.com/99205979/163348353-3bcbf4da-b17c-42f4-8138-e65b6a69f8fb.png)

Thanks in advance for your help and support.

**Comment by rquey** [comment, marked as answer]
No, discussed here: #100.

**Reply by majidkavousi** [reply]
Hi,
Thanks for your reply. The hyperlink doesn't work btw.
